shared/actions/users.tsx

```diff
@@ -13,8 +17,33 @@ const onIdentifyUpdate = (
     newlyFixed: action.payload.params.okUsernames || [],
   })
 
+const fetchUserBio = async (state: TypedState, action: Chat2Gen.SelectConversationPayload) => {
```

Comment:
i think this function should stay in chat saga, but it should just dispatch `UsersGen.createUpdateBio({username})`

this saga should have a chainAction
`chainAction2(UsersGen.updateBio, fetchUserBio)` that just makes the rpc call and fills the data